hr_attendance_schedule/static/src/js/kiosk_confirm.js

```diff
@@ -0,0 +1,33 @@
+odoo.define('hr_attendance_schedule.kiosk_confirm', function(require){
+  "use strict";
+
+  var Model = require('web.Model');
+  var KioskConfirm = require('hr_attendance.kiosk_confirm');
+
+  KioskConfirm.include({
+    start_clock: function() {
+      var self = this;
+      this.$('.o_hr_attendance_clock').text("...");
+      var server_clock = new Model('hr_attendance_schedule.clock');
+      server_clock.call('get_system_clock')
```

Comment:
it's a good habit to return the deferred object(s) of asynchronous calls made in functions, so that other modules can attach to them